src/phantom.js

```diff
@@ -30,8 +31,9 @@ export default class Phantom {
             throw new Error('Unexpected type of parameters. Expecting args to be array.');
         }
 
-        logger.debug(`Starting ${phantomjs.path} ${args.concat([__dirname + '/shim.js']).join(' ')}`);
-        this.process = spawn(phantomjs.path, args.concat([__dirname + '/shim.js']));
+        let pathToShim = path.normalize(__dirname + '/shim.js');
```

Comment:
@ZiggyQubert was this broken before? I think it's fine to merge it in, but I am curious why you had to do this.